### PR TITLE
Activity improvement

### LINKED ISSUE #16: Improve "Inschrijvingen" overview for activities
The "Inschrijvingen" overview that shows when you click on the persons icon is not very convenient. You have to click on an arrow to see the registration details. But what if you want a quick overview of all the registration details (for example, to count how many people are vegetarian)? It would be ideal to just show one table with all of the registrants (their name, email, phone perhaps) and their answers to the registration questions (e.g. vegetarian, sailing level)

### PR DESCRIPTION
**Pull request by I-Stork**
## Description
This PR contains two fairly convenient changes:
- It shows registrations in a table, which gives a better overview of all data.
- It allows members from the board to unregister participants (delete registrations).

### Checklist
- [ ] Meets acceptance criteria
- [ ] Added tests
- [ ] All tests are working
- [ ] Fixed linting warnings

### CODE REVIEW

**Comment by I-Stork** [comment]
Lets wait with this PR until export has been pushed. Seems like a very useful functionality to add

File: src/app/activities/activities.service.ts [hunk 1]
```diff
@@ -39,17 +43,30 @@ export class ActivitiesService {
...
+      })
+    );
   }
 
   public getRegistration(id: string): Observable<Registration> {
     return this.http.get<Registration>(`${environment.apiUrl}/activity-registrations/${id}`);
   }
+
+  public deleteRegistration(id: string): Observable<void> {
+    console.log("Deleting from service: " + id);
```
Comment (dutchmartin) [inline src/app/activities/activities.service.ts:60]: Please delete console log
```suggestion
```

**Review by dutchmartin — COMMENTED** [review]
Fix that nitpick and merge this!
Tested it, and it works great. Awesome job!

**Comment by I-Stork** [comment]
Merged!

**Pull request closed without merging**